### PR TITLE
Remove another instance of crossing boundary while in transaction.

### PR DESCRIPTION
**Pull request by jgrund**
Signed-off-by: Joe Grund <jgrund@whamcloud.io>

### CODE REVIEW

File: chroma_core/services/lustre_audit/update_scan.py [hunk 1]
```diff
@@ -218,17 +218,19 @@ def update_target_mounts(self):
                         ["mounted", "unmounted"],
                     )
 
-            if target_mount.target.active_mount == None:
-                TargetRecoveryInfo.update(target_mount.target, {})
-                TargetRecoveryAlert.notify(target_mount.target, False)
+            if target_mount.target.active_mount is None:
```
Comment (utopiabound) [inline chroma_core/services/lustre_audit/update_scan.py]: Should this check be inside the atomic transaction?

Comment (jgrund) [reply chroma_core/services/lustre_audit/update_scan.py]: Like this?

```python
with transaction.atomic():
    if target_mount.target.active_mount is None:
        TargetRecoveryInfo.update(target_mount.target, {})
        TargetRecoveryAlert.notify(target_mount.target, False)
    elif mounted_locally:
        recovering = TargetRecoveryInfo.update(target_mount.target, recovery_status)
        TargetRecoveryAlert.notify(target_mount.target, recovering)
```

Comment (utopiabound) [reply chroma_core/services/lustre_audit/update_scan.py]: I think so.

Comment (jgrund) [reply chroma_core/services/lustre_audit/update_scan.py]: Sure, I'll update.

**Pull request merged**